packages/grafana-ui/src/components/TimePicker/TimeRangePicker/TimePickerFooter.tsx

```diff
@@ -45,7 +45,7 @@ export const TimePickerFooter: FC<Props> = props => {
   if (isEditing) {
     return (
       <div className={cx(style.container, style.editContainer)}>
-        <div className={style.timeZoneContainer}>
+        <div aria-label="Time zone picker select container" className={style.timeZoneContainer}>
```

Comment:
Shouldn't we keep these in e2e selectors?

Reply:
You're right. I wasn't aware that we were importing them into the app. It does make sense to.